### ISSUE TITLE
How to resize the height of the PopupDialog?

### ISSUE THREAD

**Issue by katunch** [Orderella/PopupDialog#289]
Hi

Is there a way to resize the height of the PopupDialog? 

For example: I have an initial height of 300. There is a textfield inside the popup. The popup should now grow with the amount of lines which i write into the textfield. 

Is there any easy way for that?

Thx

**Comment by mwfire** [comment]
Hey there @katunch,

sry for my late reply! The height should adapt whenever the height of the custom view changes. You just have to make sure that constraints from top to bottom are there, so the height can be inferred.

Does that work for you?

**Comment by kokitang** [comment]
> Hey there @katunch,
> 
> sry for my late reply! The height should adapt whenever the height of the custom view changes. You just have to make sure that constraints from top to bottom are there, so the height can be inferred.
> 
> Does that work for you?

seems not work.. could you make an example showing this behavior?

**Comment by mwfire** [comment]
![bildschirmaufnahme 2018-12-02 um 18 19 29 2018-12-02 18_32_10](https://user-images.githubusercontent.com/1749092/49342861-a1fff680-f660-11e8-8838-f2514d2bf6b7.gif)

I altered the example project a bit, that is, I created an outlet for the top constraint in the rating view controller. When changing the constant (on textfield focus / unfocus), the height is adapted automatically. I did not check with a multiline text, though.
